blog/2020-11-05-HowMuchFasterIsJava15.adoc

```diff
@@ -0,0 +1,105 @@
+= How much faster is Java 15?
+:page-interpolate: true
+:awestruct-author: mtomco
+:awestruct-layout: blogPostBase
+:awestruct-tags: [production, benchmark]
+
+Java 15 was released on the 15th of September 2020 and has promised to bring with it a few performance tweaks in its G1GC
+and ParallelGC garbage collectors. What does it mean for
+https://www.optaplanner.org/[OptaPlanner]? Have the improvements made in the G1GC outperform the reigning champion since Java 8 or does the ParallelGC still
+reign supreme on most use-cases? Is ParallelGC even faster than it was the previous time? Let's put it to the test!
```

Comment:
Is the angle of this story purely the GC's against each other?
A great angle is also what benefits we get from upgrading the JDK from 11 to 15 (=> free performance), regardless of the GC.